docs/book/in-depth-guide/data-binding.md

```diff
@@ -317,10 +317,10 @@ The three differences between the `add` and `edit` templates are:
 Because the URI requires the identifier, we need to ensure the identifier is
 passed. The way we've done this in the controllers is to pass the identifier as
 a parameter: `$this->url('blog/edit/', ['id' => $id])`. This would require that
-we pass the original `Post` instance or the identifier we pull from it to the
-view, however. laminas-router allows another option, however: you can tell it to
-re-use currently matched parameters.  This is done by setting the last parameter
-of the view-helper to `true`: `$this->url('blog/edit', [], true)`.
+we pass the original `Post` instance (or the identifier we pull from it) to
+the view. However, laminas-router allows another option: we can tell it to
+re-use the currently matched parameters. This is done by setting the last
+parameter of the view-helper to `true`: `$this->url('blog/edit', [], true)`.
```

Comment:
These changes are unrelated to the current topic of this pull request, please remove all theses changes. If the correction is still relevant then please create a new pull request.

Reply:
@froschdesign I have reverted those changes as requested. I will be making a separate PR for them because I feel that paragraph is a bit difficult to read.